using_openshift/authorization.adoc

```diff
@@ -10,78 +10,99 @@
 toc::[]
 
 == Overview
-*Authorization is still in the experimental stage, so the API, CLI, and rule expression are subject to change or removal without notice.*
+*Authorization is still in the experimental stage, so the API, command line, and rule expressions are subject to change or removal without notice.*
 
-Authorization happens as a separate step from authentication.  Authentication determines who (identity) is taking the action, but authorizations determines whether that identity is allowed to perform the action.
+Authorization determines whether an identity is allowed to perform any action. Note that authorization is a separate step from authentication, which is more about determining who (identity) is taking the action.
 
-Authorization is managed using rules, roles, and bindings.  A rule grants or denies the ability to perform an action.  A role is a collection of rules.  A binding associates a role with a list of users and groups.  Users and groups may be bound to multiple roles at the same time.
+Authorization is managed using rules, roles, and bindings. A rule determines who has the ability to perform an action, and a role is a collection of rules. Users and groups may be bound to multiple roles at the same time. Lastly, a binding associates a role with a list of users and groups. 
```

Comment:
A rule doesn't determine who has the ability to perform an action.  The rule describes the permission, but it is the binding that decides who the rule is applied to.